### DISCUSSION TITLE
Option to add request color tagging

### DISCUSSION THREAD

**Discussion by wotta** [ngrok/ngrok#64] [Ideas]
Hello everyone,

I am wondering if it would be possible to add color tagging for requests that have come in. Currently, I am debugging 2 requests that have caused weird behavior and while doing that investigating I am gathering more information which means the data keeps being sent in through ngrok which will clutter the interface after some time.

Having the option to color tag in some way would make it much easier for me to know which requests are of high interest for me to follow up on. The example below is done with a simple background color style added to it which would solve my issue at hand directly.

Let me know if this is something that could be implemented and if I would have an option to do so in order to not burden the people on this.

![ngrok screenshot showing color tagged request rows](https://github.com/user-attachments/assets/a2399ddc-9c3a-44e9-accf-997873b961f7)

**Comment by russorat** [comment]
@wotta thanks for the discussion. It's unlikely that we will update the local inspection interface in the agent, but I've sent this idea to the team working on https://dashboard.ngrok.com/traffic-inspector to see what they think.

**Reply by wotta** [reply]
Thanks for passing it along and have a nice day!
